thirdparty/basis_universal/encoder/basisu_comp.cpp

```diff
@@ -3478,7 +3478,7 @@ namespace basisu
 				// TODO: Remove this flag entirely
 				params.m_multithreading = true; 
 				
-				// Stop using OpenCL if a failure ever occurs.
+				// Stop using OpenCL if a failure ever occurrs.
```

Comment:
```suggestion
				// Stop using OpenCL if a failure ever occurs.
```
I don't think this one is right :)